packages/core/helper-plugin/src/features/GuidedTour.tsx

```diff
@@ -27,15 +27,23 @@ interface GuidedTourContextValue {
       create: boolean;
       success: boolean;
     };
-    transferTokens: {
+    mediaLibrary: {
+      create: boolean;
+      success: boolean;
+    };
+    profile: {
+      create: boolean;
+      success: boolean;
+    };
+    inviteUser: {
       create: boolean;
       success: boolean;
     };
   };
-  isGuidedTourVisible: boolean;
+  guidedTourVisibility: string | null;
   isSkipped: boolean;
   setCurrentStep: (step: Step | null) => void | null;
-  setGuidedTourVisibility: (isVisible: boolean) => void;
+  setGuidedTourVisibility: (isVisible: string) => void;
```

Comment:
this doesn't make sense either `isVisible` doesn't pair to a `string` clearly. Lets revise the variable you want to add and make the code clearer, you can also use a union of strings to denote exactly the options we want to be used

Reply:
It's not really variables I want to add but rather variables that need to change to not just be: true or false but super-admin or admin. The logic right now is to either show the GT or not, what I want is to show it every time but distinguish them so I tried to adapt the code to it. 

I need to better understand why some keys can't be updated tho 🤔

Reply:
> I need to better understand why some keys can't be updated tho 

Because it's exported from the `helper-plugin` which means it can be used by anyone outside of strapi, whilst it's unlikely, it's not impossible, as such we have to preserve these APIs as is, we can add more and deprecate, but can't remove them without breaking changes. Since you want to merge this in V4 especially.